### PR TITLE
ntfyme: add common page

### PR DESCRIPTION
**Pull request by AnirudhG07**
- [X] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [X] The page(s) have at most 8 examples.
- [X] The page description(s) have links to documentation or a homepage.
- [X] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [X] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message-and-pr-title).
- **Version of the command being documented (if known):**

### CODE REVIEW

**Comment by spageektti** [comment]
Hi, thanks for your contribution. Sorry for the format of my suggestion.

**Comment by AnirudhG07** [comment]
> Hi, thanks for your contribution. Sorry for the format of my suggestion.

Thanks for the suggestion! I have added it.

**Comment by spageektti** [comment]
You should delete the last blank line. I added it by mistake.

**Review by spageektti — APPROVED** [review]
LGTM.

File: pages/common/ntfyme.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
+# ntfyme
+
+> A notification tool to track and notify you about your long-running termination process.
+> Send notifications with success/error messages with Gmail, Telegram, and more.
+> More information: <https://github.com/AnirudhG07/ntfyme>.
+
+- Directly run your command:
+
+`ntfyme exec {{-c|--cmd}} {{COMMAND}}`
```
Comment (spageektti) [inline pages/common/ntfyme.md:9]:
```suggestion
`ntfyme exec {{-c|--cmd}} {{command}}`
```

File: pages/common/ntfyme.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+- Pipe your command and run:
+
+`echo {{COMMAND}} | ntfyme exec`
```
Comment (spageektti) [inline pages/common/ntfyme.md:13]:
```suggestion
`echo {{command}} | ntfyme exec`
```

File: pages/common/ntfyme.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+- Run multiple commands by enclosing them in quotes:
+
+`echo "{{COMMAND1; COMMAND2; COMMAND3}}" | ntfyme exec`
```
Comment (spageektti) [inline pages/common/ntfyme.md:17]:
```suggestion
`echo "{{command1; command2; command3}}" | ntfyme exec`
```

File: pages/common/ntfyme.md [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+
+- Track and terminate the process after prolonged suspension:
+
+`ntfyme exec {{-t|--track-process}} {{-c|--cmd}} {{COMMAND}}`
```
Comment (spageektti) [inline pages/common/ntfyme.md:21]:
```suggestion
`ntfyme exec {{-t|--track-process}} {{-c|--cmd}} {{command}}`
```

**Pull request merged**